server/src/main/java/com/coincoinche/LadderController.java

```diff
@@ -0,0 +1,37 @@
+package com.coincoinche;
+
+import com.coincoinche.ratingplayer.EloPlayer;
+import com.coincoinche.repositories.UserRepository;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+/** Controller esponsible for providing the ladder (users and their rating). */
+@RestController
+public class LadderController {
+
+  private static final Logger logger = LoggerFactory.getLogger(LadderController.class);
+  private static final String DEFAULT_LADDER_SIZE = "100";
```

Comment:
Why not a number? ^^

Reply:
it seems `defaultValue` in `RequestParam` must be a `String` (even if the actual parameter is an `int`)